Applet/src/com/android/javacard/keymaster/RemotelyProvisionedComponentDevice.java

```diff
@@ -40,9 +40,11 @@ public class RemotelyProvisionedComponentDevice {
   private static final byte TRUE = 0x01;
   private static final byte FALSE = 0x00;
   // RKP Version
```

Comment:
Looks like KMFunctionalTest changes were missing. Are those changes already in. ?